Userland/Libraries/LibWeb/HTML/Parser/HTMLTokenizer.cpp

```diff
@@ -210,16 +210,18 @@ Optional<u32> HTMLTokenizer::next_code_point()
 
 void HTMLTokenizer::skip(size_t count)
 {
-    m_source_positions.append(m_source_positions.last());
+    if (!m_source_positions.is_empty())
+        m_source_positions.append(m_source_positions.last());
     for (size_t i = 0; i < count; ++i) {
         m_prev_utf8_iterator = m_utf8_iterator;
         auto code_point = *m_utf8_iterator;
         if (code_point == '\n') {
-            m_source_positions.last().column = 0;
-            m_source_positions.last().line++;
-        } else {
+            if (!m_source_positions.is_empty()) {
+                m_source_positions.last().column = 0;
+                m_source_positions.last().line++;
+            }
+        } else if (!m_source_positions.is_empty())
             m_source_positions.last().column++;
-        }
```

Comment:
Please put this back, branches need curly braces if any of the sibling branches have them.